src/types/common.ts

```diff
@@ -69,6 +69,7 @@ export const ClientResponseSchema = z.object({
   familyName: z.string(),
   email: z.string(),
   companyId: z.string(),
+  companyIds: z.array(z.string()).optional(),
```

Comment:
```suggestion
  companyIds: z.array(z.string().uuid()).optional(),
```
Company id's are uuid's right